src/Components/Filter/filter.css

```diff
@@ -0,0 +1,9 @@
+.filter {
+  border: 3px solid blue;
```

Comment:
seems you have used direct colors for to show skeleton but make sure to use variables in actual branch something repetative

Reply:
sure